10.9-libcxx/stable/main/finkinfo/libs/json-c4-shlibs.info

```diff
@@ -0,0 +1,62 @@
+### json-c 0.13 bumps to libN=5
```

Comment:
It's just a comment, but Fink standard is that libN matches the library install name, not the compatibility version. So json-c4 is correct because the library name is libjson-c.4.dylib, and libN is also '4' when we comment about having to update (or not) the package name.